src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/GlobalSuppressions.cs

```diff
@@ -120,3 +120,11 @@
 [assembly: SuppressMessage("Build", "CA2227:Change 'SourceSearchException' to be read-only by removing the property setter.", Justification = "<Pending>", Scope = "member", Target = "~P:NuGet.PackageManagement.VisualStudio.SearchResult`1.SourceSearchException")]
 [assembly: SuppressMessage("Build", "CA2227:Change 'SourceSearchStatus' to be read-only by removing the property setter.", Justification = "<Pending>", Scope = "member", Target = "~P:NuGet.PackageManagement.VisualStudio.SearchResult`1.SourceSearchStatus")]
 [assembly: SuppressMessage("Globalization", "CA1308:Normalize strings to uppercase", Justification = "We require lowercase package names in telemetry so that the hashes are consistent", Scope = "member", Target = "~M:NuGet.PackageManagement.Telemetry.VSTelemetryServiceUtility.NormalizePackageId(System.String)~System.String")]
+[assembly: SuppressMessage("Usage", "VSTHRD010:Invoke single-threaded types on Main thread", Justification = "<Pending>", Scope = "member", Target = "~M:NuGet.PackageManagement.VisualStudio.WebSiteProjectSystem.AddGacReference(System.String)")]
```

Comment:
Do we have a process to go back later and fix these for real?  This list looks really long

Reply:
In my opinion, since there's "only" 8 new analyzer warnings, they should be fixed in this PR, rather than leaving them for later. I think we set bad precedent by creating the global suppression files in the first place, but the point at the time was there were hundreds of analyzer warnings (which we choose to treat as errors because ????), so we enabled the analyzer to ensure new code didn't violate the warning, but allowing us to take time later to fix existing code.

Reply:
I think we should create an issue and link that issue in the justification.

Reply:
Added justification pointing to https://github.com/NuGet/Client.Engineering/issues/1321